### PR TITLE
add support for the "@starting-style" conditional group rules

### PR DESCRIPTION
**Pull request by bes-internal**

### CODE REVIEW

File: test/fixtures/starying-style-min.css
Comment (radium-v) [inline test/fixtures/starying-style-min.css:1]: File name should be `starting-style-min.css`

File: test/fixtures/starying-style.css
Comment (radium-v) [inline test/fixtures/starying-style.css:1]: File name should be `starting-style.css`

File: test/fixtures/starying-style-min.css
Comment (bes-internal) [reply test/fixtures/starying-style-min.css:1]: > File name should be `starting-style-min.css`
These are changes to existing files. If you want to suggest changes, open a new ticket

File: test/fixtures/starying-style.css
Comment (bes-internal) [reply test/fixtures/starying-style.css:1]: > File name should be `starting-style.css`
These are changes to existing files. If you want to suggest changes, open a new ticket

**Comment by jakubpawlowicz** [comment]
Looks good, thanks!

**Pull request merged**